Remotion/Web/Core/UI/Controls/ListMenuImplementation/Rendering/ListMenuRenderer.cs

```diff
@@ -162,7 +162,7 @@ private void RenderListMenuItem (ListMenuRenderingContext renderingContext, WebM
       bool showText = menuItem.Style == WebMenuItemStyle.Text || menuItem.Style == WebMenuItemStyle.IconAndText;
 
       renderingContext.Writer.AddAttribute (HtmlTextWriterAttribute.Id, GetMenuItemClientID (renderingContext, index));
-      renderingContext.Writer.AddAttribute (HtmlTextWriterAttribute.Class, "listMenuItem");
+      renderingContext.Writer.AddAttribute (HtmlTextWriterAttribute.Class, "listMenuItem " + CssClassDefinition.RemotionThemed);
```

Comment:
please introduce a property.

Also, am considering :
Themability of ListMenuItem and themability of BocList commands in the BocList-cells should probably be the same. Right now, I think this here is the only place where we are theming an anchor-tag. Maybe skip this for now and do it consistently for all relevant anchor-tags later on. Please make a follow-up task and put it in september backlog sprint.